lib/bloc/lnurl/lnurl_actions.dart

```diff
@@ -28,3 +28,9 @@ class Login extends AsyncAction {
 
   Login(this.response, {this.jwt = false});
 }
+
+class Pay extends AsyncAction {
```

Comment:
This action is used to fetch the invoice on second stage. Maybe use better naming? "FetchInvoice" or something similar?